assets/javascripts/kitten/components/layout/horizontal-stroke/common/stroke-modifier-styles.js

```diff
@@ -0,0 +1,186 @@
+import { css } from 'styled-components'
+import { pxToRem } from '../../../../helpers/utils/typography'
+import { ScreenConfig } from '../../../../constants/screen-config'
+
+const strokeModifiers = [
+  {
+    name: 'primary',
+    mobile: {
+      width: 40,
+      height: 6,
+      top: 20,
+      bottom: 40,
+    },
+    tablet: {
+      width: 50,
+      height: 6,
+      top: 20,
+      bottom: 40,
+    },
+    desktop: {
+      width: 50,
+      height: 6,
+      top: 20,
+      bottom: 40,
+    },
+  },
+
+  {
+    name: 'secondary',
+    mobile: {
+      width: 30,
+      height: 4,
+      top: 15,
+      bottom: 30,
+    },
+    tablet: {
+      width: 40,
+      height: 6,
+      top: 20,
+      bottom: 40,
+    },
+    desktop: {
+      width: 50,
+      height: 6,
+      top: 20,
+      bottom: 40,
+    },
+  },
+
+  {
+    name: 'tertiary',
+    mobile: {
+      width: 30,
+      height: 4,
+      top: 15,
+      bottom: 30,
+    },
+    tablet: {
+      width: 30,
+      height: 4,
+      top: 15,
+      bottom: 30,
+    },
+    desktop: {
+      width: 40,
+      height: 6,
+      top: 20,
+      bottom: 40,
+    },
+  },
+
+  {
+    name: 'quaternary',
+    mobile: {
+      width: 30,
+      height: 4,
+      top: 15,
+      bottom: 30,
+    },
+    tablet: {
+      width: 30,
+      height: 4,
+      top: 15,
+      bottom: 30,
+    },
+    desktop: {
+      width: 30,
+      height: 4,
+      top: 15,
+      bottom: 30,
+    },
+  },
+
+  {
+    name: 'quinary',
+    mobile: {
+      width: 30,
+      height: 4,
+      top: 15,
+      bottom: 30,
+    },
+    tablet: {
+      width: 30,
+      height: 4,
+      top: 15,
+      bottom: 30,
+    },
+    desktop: {
+      width:0,
```

Comment:
```suggestion
      width: 0,
```

Prettier est passé dans le fichier ? Cette modif me met le doute.